pandas/sparse/series.py

```diff
@@ -630,21 +630,30 @@ def take(self, indices, axis=0, convert=True, *args, **kwargs):
 
     def cumsum(self, axis=0, *args, **kwargs):
         """
-        Cumulative sum of values. Preserves locations of NaN values
+        Cumulative sum of non-NA/null values.
+
+        When performing the cumulative summation, any non-NA/null values will
+        be skipped. The resulting SparseSeries will preserve the locations of
+        NaN values, but the fill value will be `np.nan` regardless.
+
+        Parameters
+        ----------
+        axis : int
+            Axis over which to perform the cumulative summation. This
+            parameter is ignored because `SparseSeries` is 1-D, but it
+            is kept in the signature for consistency with other sparse
+            array-like objects.
 
         Returns
         -------
-        cumsum : SparseSeries if `self` has a null `fill_value` and a
-                 generic Series otherwise
+        cumsum : SparseSeries
         """
         nv.validate_cumsum(args, kwargs)
```

Comment:
Fair enough, though it was misleading because I had initially assumed the `axis` parameter did something, even though it doesn't do anything in this implementation.